alerts/apps_alerts.libsonnet

```
@@ -6,11 +6,35 @@
     prefixedNamespaceSelector: if self.namespaceSelector != null then self.namespaceSelector + ',' else '',
   },
 
+  // TODO move into libs
+  local wrap_rule_for_labels(rule) =
+    // Detect Kind of rule from name unless hidden `kind field is passed in the rule`
+    local kind =
+      if std.objectHas(rule, 'kind') then rule.kind
```

if `kind` is a hidden field, you need to use `std.objectHasAll`.
Or, alternatively you can use the `in` operator:

```suggestion
      if 'kind' in rule then rule.kind
```